e2e/support/config.js

```diff
@@ -36,6 +36,15 @@ const defaultConfig = {
      **                        PREPROCESSOR                            **
      ********************************************************************/
 
+
+     if (runWithReplay) {
+      replay.default(on, config, {
+        upload: true,
+        apiKey: process.env.REPLAY_API_KEY,
+      });
+      on = replay.wrapOn(on);
```

Comment:
This should be before the replay plug-in

Reply:
Thanks for catching it @ryanjduffy 
Fixed in https://github.com/metabase/metabase/pull/36907/commits/48eed4ae8299216806b8c513e7cc30c2206b325b